### PR TITLE
Add Glacite Tunnels info to /pv

### PR DESCRIPTION
**Pull request by nea89o**

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/hotm/HotmTreeScreen.kt [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+            .registerTypeAdapterFactory(LispProgramSerializer)
+            .create()
+
+        var renderer: HotmTreeRenderer? = null
+
+        @SubscribeEvent
+        fun onRepoReload(event: RepositoryReloadEvent) {
+            renderer = runCatching {
+                val hotmLayoutFile = NotEnoughUpdates.INSTANCE.manager.repoLocation
+                    .resolve("constants/hotmlayout.json")
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/hotm/HotmTreeScreen.kt:69]: Why isnt this done the normal way in Constants.java?

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]

**Review by NopoTheGamer — COMMENTED** [review]
image
probably shouldn't show overflow level, not sure if this happened before this pr

**Comment by jani270** [comment]
> image probably shouldn't show overflow level, not sure if this happened before this pr

As seen in the git diff, the hotm part has not been changed by us.

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/hotm/HotmTreeScreen.kt (hunk 1 above)
Comment (nea89o) [reply src/main/java/io/github/moulberry/notenoughupdates/profileviewer/hotm/HotmTreeScreen.kt:69]: The normal way in Constants.java is bad. It just loads things into a JsonObject (which we shouldn't use raw like that anyway). There is just not much of a point in doing that when i will just load it into my own data structure anyways.

**Review by NopoTheGamer — APPROVED** [review]

**Pull request merged**